docs/token-contracts/admin-lambda.md

```diff
@@ -0,0 +1,386 @@
+- The `lambda` gets `EXEC`uted if and only if `SENDER` is equal to the current administrator's address
+
+
+### Origination
+
+```bash
+$ tezos-client --wait none originate contract AdminLambda \
+  transferring 0 from $BOB_ADDRESS running \
+  "$(curl https://raw.githubusercontent.com/tqtezos/admin-lambda/012f6a0b3515bcdb223ee109478e6146a26f09cc/admin_lambda.tz)" \
+  --init "\"$BOB_ADDRESS\"" --burn-cap 0.354
+
+  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
+                                 Dload  Upload   Total   Spent    Left  Speed
+100   224  100   224    0     0   2382      0 --:--:-- --:--:-- --:--:--  2382
+Waiting for the node to be bootstrapped before injection...
+Current head: BKq3SE7psZ1H (timestamp: 2020-10-20T19:18:05-00:00, validation: 2020-10-20T19:18:34-00:00)
+Node is bootstrapped, ready for injecting operations.
+Estimated gas: 12274 units (will add 100 for safety)
+Estimated storage: 354 bytes added (will add 20 for safety)
+Operation successfully injected in the node.
+Operation hash is 'ooLMGjrMMiKCxNXJDtVqeYfqcQZi3DA6Atcw2cFGmuDBvKqNcpC'
+NOT waiting for the operation to be included.
+Use command
+  tezos-client wait for ooLMGjrMMiKCxNXJDtVqeYfqcQZi3DA6Atcw2cFGmuDBvKqNcpC to be included --confirmations 30 --branch BLLHBu4Gt9g9zMP8hNPamdh3kcvgzLHrKVGzn2UGF5kXPp7ti1n
+and/or an external block explorer to make sure that it has been included.
+This sequence of operations was run:
+  Manager signed operations:
+    From: tz1bDCu64RmcpWahdn9bWrDMi6cu7mXZynHm
+    Fee to the baker: ꜩ0.001578
+    Expected counter: 624033
+    Gas limit: 12374
+    Storage limit: 374 bytes
+    Balance updates:
+      tz1bDCu64RmcpWahdn9bWrDMi6cu7mXZynHm ............. -ꜩ0.001578
+      fees(tz1Ke2h7sDdakHJQh8WX4Z372du1KChsksyU,400) ... +ꜩ0.001578
+    Origination:
+      From: tz1bDCu64RmcpWahdn9bWrDMi6cu7mXZynHm
+      Credit: ꜩ0
+      Script:
+        { parameter (lambda address (pair (list operation) address)) ;
+          storage address ;
+          code { DUP ;
+                 CAR ;
+                 SWAP ;
+                 CDR ;
+                 DUP ;
+                 SENDER ;
+                 COMPARE ;
+                 EQ ;
+                 IF { EXEC } { FAILWITH } } }
+        Initial storage: "tz1bDCu64RmcpWahdn9bWrDMi6cu7mXZynHm"
+        No delegate for this contract
+        This origination was successfully applied
+        Originated contracts:
+          KT1JPmnRk2v9y9MEyCwges5is2reucDFgM7n
+        Storage size: 97 bytes
+        Paid storage size diff: 97 bytes
+        Consumed gas: 12274
+        Balance updates:
+          tz1bDCu64RmcpWahdn9bWrDMi6cu7mXZynHm ... -ꜩ0.097
+          tz1bDCu64RmcpWahdn9bWrDMi6cu7mXZynHm ... -ꜩ0.257
+
+New contract KT1JPmnRk2v9y9MEyCwges5is2reucDFgM7n originated.
+Contract memorized as AdminLambda.
+```
+
+Make a `bash` variable for it:
+
+```bash
+ADMIN_LAMBDA='KT1JPmnRk2v9y9MEyCwges5is2reucDFgM7n'
+```
+
+### Originate a `FA2`
+
```

Comment:
This is a good spot to put a summary of what the whole section does: using AdminLambda to emulate allowances:
- bob has N tokens
- it want to allow Fred to spend M tokens on his behalf
...